Formula/pwntools.rb

```diff
@@ -159,12 +158,13 @@ class Pwntools < Formula
   end
 
   resource "urllib3" do
-    url "https://files.pythonhosted.org/packages/d7/8d/7ee68c6b48e1ec8d41198f694ecdc15f7596356f2ff8e6b1420300cf5db3/urllib3-1.26.3.tar.gz"
-    sha256 "de3eedaad74a2683334e282005cd8d7f22f4d55fa690a2a1020a416cb0a47e73"
+    url "https://files.pythonhosted.org/packages/cb/cf/871177f1fc795c6c10787bc0e1f27bb6cf7b81dbde399fd35860472cecbc/urllib3-1.26.4.tar.gz"
+    sha256 "e7b021f7241115872f92f43c6508082facffbd1c048e3c6e2bb9c2a157e28937"
   end
 
   def install
     virtualenv_install_with_resources
+    Dir[libexec/"**/*"].each { |file| rm_f file if File.zero?(file) }
```

Comment:
This is risky as it will delete `__init__.py` files.

Reply:
Would be useful to expose the allowlist to the Formula class so it can be used to check whether files can be deleted.